src/components/EntityInfo/EntityMenu.vue

```diff
@@ -168,17 +168,21 @@
                 content-class="right-tooltip"
               >
                 <template #activator="{ on }">
-                  <v-list-item
+                  <div
                     v-if="enableAgmExtension"
-                    id="agm-ext-list-item"
-                    :disabled="!isAllowed(AllowableActions.AGM_EXTENSION)"
                     v-on="on"
-                    @click="goToAgmExtensionFiling()"
                   >
-                    <v-list-item-title>
-                      <span class="app-blue">Request AGM Extension</span>
-                    </v-list-item-title>
-                  </v-list-item>
+                    <v-list-item
+                      id="agm-ext-list-item"
+                      :disabled="!isAllowed(AllowableActions.AGM_EXTENSION)"
+                      v-on="on"
+                      @click="goToAgmExtensionFiling()"
+                    >
+                      <v-list-item-title>
+                        <span class="app-blue">Request AGM Extension</span>
+                      </v-list-item-title>
+                    </v-list-item>
+                  </div>
```

Comment:
Wrapping the `v-list-item` in a div with a `v-if` appears to fix bcgov/entity#18680

Reply:
Is it the same template structure as for the other AGM filing?

Reply:
It is the same as for AGM Location Change, which also uses a custom tool tip message depending on whether the action is allowed or not.

Reply:
Weird.

Anyways, LGTM.